sdk/spring/spring-messaging-azure-eventhubs/src/main/java/com/azure/spring/messaging/eventhubs/core/listener/EventHubsMessageListenerContainer.java

```diff
@@ -62,7 +57,9 @@ protected void doStart() {
 
     @Override
     protected void doStop() {
-        this.delegate.stop();
+        if (this.delegate != null) {
```

Comment:
I am not sure about this, but the "spotbug" hints a low risk of defect.  Seems just remove the logger in this file.